ci-operator/config/open-cluster-management/management-ingress/open-cluster-management-management-ingress-main.yaml

```diff
@@ -25,6 +25,16 @@ tests:
   secrets:
   - mount_path: /etc/sonarcloud/
     name: acm-sonarcloud-token
+- as: sonar-post-submit
```

Comment:
https://github.com/openshift/release/pull/20235/files#diff-5605f77d6ff5355084064ab476487de7b35e3e45213dfd3c9696ac95e8028815R19-R27 already has sonarcloud, can we reuse this?

Reply:
No. Sonar need to run on the PR (the pre-submit phase) and after the PR is merged (the post-submit phase).

The second sonar scan has `postsubmit: true` to run it in post-submit. The first scan does not have that setting, so it run in the pre-submit phase.